lib/table_of_contents/configuration.rb

```diff
@@ -14,7 +15,8 @@ class Configuration
         'list_class' => 'section-nav',
         'sublist_class' => '',
         'item_class' => 'toc-entry',
-        'item_prefix' => 'toc-'
+        'item_prefix' => 'toc-',
+        'use_ordered_list' => false
```

Comment:
I prefer just using `ordered_list` (boolean).

```suggestion
        'ordered_list' => false
```